lua/compe_latex_symbols/init.lua

```diff
@@ -2,6 +2,14 @@ local compe = require 'compe'
 
 local Source = {}
 
+local function copytable(orig)
```

Comment:
There's `vim.deepcopy()` function in NeoVim's Lua „standard library“, no need to define it here.